### ISSUE TITLE
Homework 2 grade

### ISSUE THREAD

**Issue by jarretrt** [adeniro21/Machine-Learning-I-DeNiro-Anthony#2]
Hey Anthony, 

There are a couple of issues with your submission that need to be fixed. Check the correlation matrix in the text again, since it's slightly different than yours. Also, your final figure showing the training and test error rates isn't quite right. There's at least one error in your "error_ridge" function. 

Best,
Ryan

**Comment by adeniro21** [comment]
Hi Ryan,

Sorry about the delay, but I have resubmitted homework 2. I updated the correlation matrix so it only contains the predictor variables from the training data set, so they now should match the correlation coefficients in the book. I also fixed an error in the error_ridge function so that the loss function is using the correct truth data from y_out, lpsa. 

Thanks,
Anthony DeNiro

**Comment by jarretrt** [comment]
Hey Anthony, 

It all looks good now. I've passed you for the assignment. 

Thanks,
Ryan
